commit ae7156ddacbf562442e5b0924395f5ef58e901d4
Author: Sergei Trofimovich <siarheit@google.com>
Date:   Tue Oct 27 23:00:33 2015 +0000

    x86 codegen: don't generate location comments
    
    The following source snippet 'module A where x */* y = 42'
    when being compiled with '-g' option emits syntactically
    invalid comment for GNU as:
    
        .text
            .align 8
            .loc 1 3 1 /* */* */
    
    Fixed by not emitting comments at all. We already suppress
    al asm comments in 'X86/Ppr.hs'.
    
    Signed-off-by: Sergei Trofimovich <siarheit@google.com>

diff --git a/compiler/nativeGen/X86/Ppr.hs b/compiler/nativeGen/X86/Ppr.hs
index 3fe1695..db2a955 100644
--- a/compiler/nativeGen/X86/Ppr.hs
+++ b/compiler/nativeGen/X86/Ppr.hs
@@ -506,9 +506,8 @@ pprInstr (COMMENT _) = empty -- nuke 'em
 pprInstr (COMMENT s) = ptext (sLit "# ") <> ftext s
 -}
 
-pprInstr (LOCATION file line col name)
-   = ptext (sLit "\t.loc ") <> ppr file <+> ppr line <+> ppr col <>
-     ptext (sLit " /* ") <> text name <> ptext (sLit " */")
+pprInstr (LOCATION file line col _name)
+   = ptext (sLit "\t.loc ") <> ppr file <+> ppr line <+> ppr col
 
 pprInstr (DELTA d)
    = pprInstr (COMMENT (mkFastString ("\tdelta = " ++ show d)))
